src/inet/UDPEndPointImplSockets.cpp

```diff
@@ -771,16 +771,16 @@ CHIP_ERROR UDPEndPointImplSockets::IPv4JoinLeaveMulticastGroupImpl(InterfaceId a
     }
     VerifyOrReturnError(lInterfaceAddressFound, INET_ERROR_ADDRESS_NOT_FOUND);
 
-    struct ip_mreq lMulticastRequest;
-    memset(&lMulticastRequest, 0, sizeof(lMulticastRequest));
-    lMulticastRequest.imr_interface = lInterfaceAddress.ToIPv4();
-    lMulticastRequest.imr_multiaddr = aAddress.ToIPv4();
+    // struct ip_mreq lMulticastRequest;
```

Comment:
Those comment blocks, are part of work in progress, right?

Reply:
Fixed.